src/shared/components/annotation/images/oncogenic-black.svg

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
@alisman this is a copy of the oncogenic-only.svg except changing the color. This is not the best implementation. The optional is making the icon as font and included?

Reply:
I think you can do this with css.  Support is not perfect, but good enough I think.   I tried it out and it works.  The following makes image black.

filter: brightness(0.1)